components/UI/inputHelper.tsx

```diff
@@ -0,0 +1,45 @@
+import React, { FunctionComponent, ReactNode } from "react";
+import { Tooltip, TooltipProps, styled, tooltipClasses } from "@mui/material";
+import InfoIcon from "./iconsComponents/icons/infoIcon";
+
+type InputHelperProps = {
+  helperText?: string;
+  children?: ReactNode;
+  error?: boolean;
```

Comment:
Why are all these props not mandatory ? === Why do you us `?` each time ?

Reply:
Ah sorry, children should be mandatory, I am fixing that.
helperText is not mandatory because I think it is easier to use, especially in the textField component :
![image](https://github.com/starknet-id/app.starknet.id/assets/60229704/6c66216a-d31f-4fa6-99b6-7a5e257426a1)
![image](https://github.com/starknet-id/app.starknet.id/assets/60229704/ab1715e2-3493-4613-bbcf-98433204fc00)